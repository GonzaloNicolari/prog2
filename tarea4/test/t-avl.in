# prueba de tiempo es_AVL
tiempo_es_AVL  500000 65536 1000 10
Fin
