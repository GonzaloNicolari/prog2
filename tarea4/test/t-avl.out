1>#  prueba de tiempo es_AVL.
2>
 Construyendo el Ã¡rbol. 
 Evaluando si es AVL. 
 Liberando binario. 
3>Fin.
